docs/getting-started/oracle.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: UMA Oracle Design
-sidebar_label: Oracle Solution
+title: UMA's Oracle Design
+sidebar_label: Optimistic Oracle Service
```

Comment:
This doesn't cover the optimistic oracle yet - do we want to leave as is until this is switched from the DVM to oo?